MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyTitle>MoreLinq.Test</AssemblyTitle>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <Authors>MoreLINQ Developers</Authors>
+    <TargetFrameworks>netcoreapp1.0;net451</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType>portable</DebugType>
+    <AssemblyName>MoreLinq.Test</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>MoreLinq.Test</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PackageTags>linq;extensions</PackageTags>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">$(PackageTargetFallback);dotnet5.4;portable-net451+win8</PackageTargetFallback>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">1.0.4</RuntimeFrameworkVersion>
```

Comment:
Oh dear. Why would the sdk version be different from the runtime version? Anyway, with that as a given, this looks correct. However, now I see that [global.json](https://github.com/morelinq/MoreLINQ/pull/262/files#diff-274660eb4b1b1d963a330b471e10f41cL7) lists 1.0.0.